apps/communications/dialer/js/keypad.js

```diff
@@ -184,10 +184,6 @@ var KeypadManager = {
                                                 this.hangUpCallFromKeypad);
     }
 
-    // According to the UX sound spec, we should use the "system" type for the
-    // dialer pad. See the attachment of the bug1068219.
-    TonePlayer.init('system');
-
```

Comment:
Because we're already setting it in index.js (as system) and in oncall.js (as telephony).